core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -128,6 +130,11 @@
 	private boolean configurationSucceeded = false;
 	private boolean inputMessageConsumedMultipleTimes=false;
 
+	public enum ExitState implements DocumentedEnum {
+		@EnumLabel("success") SUCCESS,
+		@EnumLabel("error") ERROR;
```

Comment:
Hoeft geen DocumentedEnum te zijn toch? Ik snap dat pipeLineResult dan aangepast moet worden, misschien een todo/taakje voor maken om dit in een losse pr te doen?